apps/strapi-cms/public/robots.txt

```diff
@@ -0,0 +1,3 @@
+# To prevent search engines from seeing the site altogether, uncomment the next two lines:
+# User-Agent: *
+# Disallow: /
```

Comment:
I'm not sure if this files is needed 🤔, but if it is I think we should uncomment the following lines:

```suggestion
User-Agent: *
Disallow: /
```